pkg/entities/conflict_response.go

```diff
@@ -0,0 +1,8 @@
+package entities
+
+// ConflictResponse represents the HTTP code 409 Conflict
+type ConflictResponse struct{}
```

Comment:
This domain error is tied with HTTP details. I think it should be renamed to `ErrEntityExists` or `EntityExistsErr` so that any HTTP details couldn't appear here.

Reply:
Please, also update the struct's comment to remove references to the HTTP protocol.

Reply:
Done